### ISSUE TITLE
Disable lada-cache for certains queries

### ISSUE THREAD

**Issue by f-liva** [spiritix/lada-cache#80]
lada-cache su Laravel 6 mi sta dando qualche problema e qualche volta le query non vengono invalidate e le successive interrogazioni al database ritornano dei dati incoerenti.

Vorrei quindi capire se è possibile disattivare lada-cache per determinate interrogazioni al database, eccovi un esempio:

```php
MyModel::withoutLadaCache()->updateOrCreate(...);
```

What do you think?

**Comment by spiritix** [comment]
L6 is not yet supported. We're working on it. As soon as it's released the mentioned problems should be gone. Do you think it's an actual use case that people might want to disable caching for specific queries? The library already offers an option to disable caching for specific tables.

**Comment by f-liva** [comment]
Yes, I think the package should allow user to disable it programmately for certain queries, not for certain tables. There are operations I would do without pass trough lada-cache

**Comment by spiritix** [comment]
I am not sure if this could be implemented like you suggested, since there are many different ways to initiate a query in Laravel (from Models, from Eloquent, from Query Builder, etc.). Possibly we'd end up having something like this:

`
LadaCache::disable();
MyModel::orderBy('id')->get();
LadaCache::enable();
`

**Comment by f-liva** [comment]
Yes this should be pefect!

**Comment by grishmadoshi** [comment]
Hello, I have similar issue.
For one of the table , there is one counter field which is updated frequently and clearing cache. So is this good idea to disable cache for it as suggested here. Can that have any impact on query where data are actually being read?

**Comment by vonec** [comment]
any updates on this feature, it be great if we can implement 

LadaCache::disable();
MyModel::orderBy('id')->get();
LadaCache::enable();

**Comment by spiritix** [comment]
Unfortunately I don't have the capacity to implement this at the moment. Contributions are welcome.

**Comment by liangguohuan** [comment]
This would help，but，it's better to be supported in lada-cache package

// disable cache all
 \config(['lada-cache.active' => false]);
// disable specific tables
\config(['lada-cache.exclude-tables' => ['skus']]);
// clear specific model cache
app()->make('lada.handler')->setBuilder((new Sku())->getQuery())->invalidateQuery(Reflector::QUERY_TYPE_UPDATE);

**Comment by zgetro** [comment]
@spiritix 

for such queries don't dependent on model eloquuent methods instad use row queries like 
``` DB::Raw( .... ) ```
[raw-expressions](https://laravel.com/docs/12.x/queries#raw-expressions)

Any of other solutions will not going to CPU optimize as it CPU require affert to update app instance in memory for request life cycle.
also `exclude-tables` is not working as expected.
